packages/core/src/node/initRsbuild.ts

```diff
@@ -156,7 +156,10 @@ async function createInternalBuildConfig(
                 config: {
                   ...tailwindConfig,
                   content: [
-                    path.join(PACKAGE_ROOT, 'src', 'theme-default', '**/*'),
+                    path.resolve(
+                      PACKAGE_ROOT,
+                      '../theme-default/src/**/*.{ts,tsx}',
```

Comment:
There are some `@apply` directives in css files, which should be included in content config.

Reply:
`content` is only for html templates and js components which contains tw class names.

it's smart enough to determine which css files should be transformed as it's a postcss plugin.

- https://tailwindcss.com/docs/installation/using-postcss
- https://tailwindcss.com/docs/content-configuration

Reply:
You are right.